container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -4,21 +4,25 @@ releases:
   url: "https://bosh.io/d/github.com/cloudfoundry-incubator/bpm-release?v=1.0.0"
   sha1: "42b95d4a0d6d15dd0b0ead62418ffb56208e2307"
 - name: capi
-  version: "1.66.0"
-  url: "https://bosh.io/d/github.com/cloudfoundry/capi-release?v=1.66.0"
-  sha1: "4236495d4a047fb9396996aba3daea9e0cb3f1bf"
+  version: "1.70.0"
+  url: "https://bosh.io/d/github.com/cloudfoundry/capi-release?v=1.70.0"
+  sha1: "bf66504c6479cff3754fb1e114e7ea45767db04c"
+- name: cf-cli
+  version: "1.8.0"
+  url: "https://bosh.io/d/github.com/bosh-packages/cf-cli-release?v=1.8.0"
+  sha1: "15d0718bc82ff9394efd2c9798873a39bf7eba06"
```

Comment:
So we can drop the manual bit in the brains?

Reply:
We will have to explicitly switch over, as smoke-tests did. I would not hold the merge about this, and rather add a trello card so to not forget.

Reply:
Ref: https://trello.com/c/reOzKNER/626-switch-brain-tests-to-cf-cli-release